examples/cifar10/test.cpp

```diff
@@ -22,16 +18,14 @@ void convert_image(const std::string &imagefilename,
                    int w,
                    int h,
                    vec_t &data) {
-  cv::Mat img = cv::imread(imagefilename);
-  if (img.data == nullptr) return;  // cannot open, or it's not an image
-  cv::Mat resized;
-  cv::resize(img, resized, cv::Size(w, h), .0, .0);
-  data.resize(w * h * resized.channels(), minv);
-  for (int c = 0; c < resized.channels(); ++c) {
-    for (int y = 0; y < resized.rows; ++y) {
-      for (int x = 0; x < resized.cols; ++x) {
+  image<> img(imagefilename, image_type::rgb);
+  image<> resized = resize_image(img, w, h);
+  data.resize(w * h * 3);
```

Comment:
use this
https://github.com/tiny-dnn/tiny-dnn/blob/master/tiny_dnn/util/image.h#L271